lib/pbench/cli/agent/commands/results/push.py

```diff
@@ -24,10 +25,25 @@ def execute(self) -> int:
             self.config,
             self.logger,
         )
-        crt.copy_result_tb(
+        res = crt.copy_result_tb(
             self.context.token, self.context.access, self.context.metadata
         )
-        return 0
+
+        try:
+            msg = res.json()["message"]
+        except requests.exceptions.JSONDecodeError:
+            msg = res.text if res.text else res.reason
+
+        if res.ok:
+            if res.status_code == 200:
+                click.echo(f"{msg}", err=True)
```

Comment:
Or `click.echo(msg, err=True)` ?? 😆

Reply:
Way too simple...

Reply:
That depends...what do you want to do [_instead_ of reporting an error](https://github.com/distributed-system-analysis/pbench/pull/3348#discussion_r1140461077)?  😀

Reply:
"Never mind..."